addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -173,3 +173,13 @@
         }
     }
 }
+
+.o_preview_questions {
+    min-height: 100px;
+}
+
+.o_survey_question_view_form {
```

Comment:
To be moved in a ``survey_question_views.scss`` (this file is for surveys only normally)